docs/csharp/proto2.md

```diff
@@ -0,0 +1,184 @@
+Proto2 support in Google.Protobuf is finally here! This document outlines the new changes brought about 
+by this initial release of proto2 support. The generated code and public API associated with proto2 
```

Comment:
nit: let's change the tone to sound less like an announcement of the initial release and more as documentation that's there to stay (because I assume we will keep this doc for a while).